src/components/ReadoutGrid.tsx

```diff
@@ -1,15 +1,44 @@
-import React from "react";
+import React, { useState } from "react";
 import "styles/_readout_grid.scss";
-import Inventory from "./Inventory";
+import "styles/_dropdown.scss";
+import "styles/_containers_and_frames.scss";
+import Weapons from "./Weapons";
 import SpellRepertoire from "./SpellRepertoire";
 import InfoBlocks from "./InfoBlocks";
+import PlayerStatsReadout from "./PlayerStatsReadout";
+
+function ReadoutGrid({ monsters, weapons, spells, playerStats }) {
+  const [readout, setReadout] = useState({
+    state: "Weapons",
+  });
+
+  const options = ["Spells", "Weapons", "Apparel", "Potions", "Miscellaneous"];
 
-function ReadoutGrid({ monsters, weapons, spells }) {
   return (
     <div className="rpgui-container framed-golden" style={{ height: "513px" }}>
+      <label className="dropdown-label" htmlFor="options">
+        Select Inventory:
+      </label>
+      <select
+        className="rpgui-dropdown"
+        value={readout.state}
+        onChange={(selected) => setReadout({ state: selected.target.value })}
+      >
+        {options.map((value) => (
+          <option value={value} key={value}>
+            {value}
+          </option>
+        ))}
+      </select>
       <div className="readout-grid">
-        <Inventory weapons={weapons} />
-        <SpellRepertoire spells={spells} />
+        {
+          {
+            Weapons: <Weapons weapons={weapons} />,
+            Spells: <SpellRepertoire spells={spells} />,
+          }[readout.state]

```

Comment:
I assume this works, but you shouldn't need to do `readout.state` (or rather, you shouldn't use an object with only one field) for a `useState` variable. Just doing `readout` should work, although you'll need to change other lines, like the default and the `onChange` to just be something like `onChange={(selected) => setReadout( selected.target.value )}`

That would change this line to just `{readout}`